admin/pre-reg/utils.py

```diff
@@ -0,0 +1,19 @@
+from framework.utils import iso8601format
+from website.project.metadata.utils import serialize_meta_schema
+
+# TODO: Write and use APIv2 serializer for this
+def serialize_draft_registration(draft, auth=None):
```

Comment:
The auth kwarg is never used. This also might be better in a serializers.py unless the utils.py filename is already a pattern that exists in osf for old-style serializers.